src/pages/Feedback/Feedback.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import {
+  Grommet,
+  Anchor,
+  Box,
+  Paragraph,
+  Grid,
+  Card,
+  Heading,
+  Image,
+  Main,
+} from 'grommet';
+import {
+  ClearOption,
+  Configure,
+  Camera,
+  CloudUpload,
+  Selection,
+  BarChart,
+  Github,
+  Slack,
+} from 'grommet-icons';
+import { grommetToolsTheme } from '../../theme.js';
+import AppHeader from '../../components/AppHeader.js';
+
+const Feedback = (props) => (
+  <Grommet background="gradient" className="App" theme={grommetToolsTheme}>
+    <AppHeader />
+
+    <Box
+      align="center"
+      margin={{
+        left: 'xlarge',
+      }}
+    >
+      <Heading
+        margin={{
+          top: 'xlarge',
+          right: 'xlarge',
+          bottom: 'large',
+        }}
+        size="customLarge"
+      >
+        Feedback
+      </Heading>
+
+      <Grid
+        alignContent="center"
+        columns={{ count: 'fill', size: 'medium' }}
+        rows="auto"
+        gap="xlarge"
+      >
+        <Box>
+          <Anchor
+            color="black"
+            size="xlarge"
+            href="#"
+            icon={<Github />}
+            label="Share feedback on Github"
+          />
+
+          <Paragraph fill={true} pad={{ right: 'large' }}>
+            Submit feedback on Grommet Tools on Github. Here, you can help
+            improve the tools and connect with other Grommet Tools lovers.
+          </Paragraph>
+
+          <Box pad={{ right: 'large' }}>
+            <Grid
+              columns="small"
+              gap={{ row: 'medium', column: 'medium' }}
+              rows="auto"
+            >
+              <Card width="medium" background="white">
+                <Anchor
+                  color="black"
+                  alignSelf="center"
+                  margin="xsmall"
+                  href="#"
+                  icon={<ClearOption />}
+                  label="Designer / Github"
+                />
+              </Card>
+              <Card width="medium" background="white">
+                <Anchor
+                  color="black"
+                  alignSelf="center"
+                  margin="xsmall"
+                  href="#"
+                  icon={<CloudUpload />}
+                  label="Publisher / Github"
+                />
+              </Card>
+              <Card width="medium" background="white">
```

Comment:
Try to see if you can make this code with fewer duplications by using a 'map' on an icon array, and the map will generate Card with a new icon on every iteration.

Reply:
Changed code to use map